### DISCUSSION TITLE
Adjust margins or padding of editor

### DISCUSSION THREAD

**Discussion by ghost** [gsantner/markor#2111] [Ideas] [answered]
The editor has great adjustments that make it easy to use for general reading, the only text adjustment really missing is the ability to change the margin padding — so we can maximize screen space.

Thx!

**Comment by gsantner** [comment, marked as answer]
Sorry I don't quite understand. Can you please elaborate, maybe with a screenshot...circleing things you would like to be smaller? Not sure if your effective question is about letting spacing.

**Reply by ghost** [reply]
Specifically the left and right margins and padding of the text editor. Pic added. It's not a lot of extra space but every pixel on a phone helps 😃

![Screenshot_20230928-153425](https://github.com/gsantner/markor/assets/103839039/ce5e82f7-7bc6-4a78-b2a1-9050e41bf144)

**Comment by guanglinn** [comment]
I made some minor adjustments to margin and padding. #2118
